docs/users/oauth_server.md

```diff
@@ -148,7 +148,8 @@ php bin/console league:oauth2-server:create-client 'Example OAuth2 Client' examp
 ```
 
 !!! note
-   You can call `--redirect-uri` multiple times.
+
```

Comment:
It's not fixed:
![image](https://github.com/ibexa/documentation-developer/assets/117284672/29772682-9999-4bd5-a631-8772dec0ed42)